packages/sdk-codegen/src/kotlin.gen.ts

```diff
@@ -236,11 +236,10 @@ ${note}
   }
 
   encodePathParams(indent: string, method: IMethod) {
-    const bump = indent + this.indentStr
```

Comment:
this was outdated code, and should have been 
```ts
const bump = this.bumper(indent)
```

I'll have to think a bit more about using `indent` rather than the `bump indent` pattern. Indent may be getting used wrong here.